### DISCUSSION TITLE
启动绑定公网ip 失败 akka Can't assign requested address

### DISCUSSION THREAD

**Discussion by lxb56** [PowerJob/PowerJob#439] [General]
鉴于群众对于公网、非局域网部署的呼声过于强烈，十分民主的本人目前放开了这个限制，用户可通过 JVM 启动参数 `-Dpowerjob.network.local.address` 来直接绑定某个IP。当然，如果要真正实现公网集群调度，免不了需要开发者自行实现启动脚本，比如先用 SHELL 获取真正的外网 IP，然后再将其作为 JVM 启动参数传给 PowerJob。

指定ip启动报错
[oms-server-akka.actor.default-dispatcher-5] WARN akka.remote.RemoteActorRefProvider - Akka Cluster not in use - Using Akka Cluster is recommended if you need remote watch and deploy.
16:55:54.931 [oms-server-akka.actor.default-dispatcher-5] ERROR akka.io.TcpListener - Bind failed for TCP channel on endpoint [/ip:10086]
java.net.BindException: [/81.68.98.68:10086] Can't assign requested address

**Comment by KFCFans** [comment]
嗯，公网调度这块目前还有设计不到位的地方，后续版本会想办法解决，暂时没办法直接绑定公网IP
